flowfile_core/flowfile_core/flowfile/code_generator/custom_node_handlers.py

```
@@ -0,0 +1,183 @@
+        settings = node.setting_input
+        class_name = custom_node_class.__name__
+        settings_dict = getattr(settings, "settings", {}) or {}
+
+        needs_collect, needs_lazy = self._check_process_method_signature(custom_node_class)
+
+        _node_name_field = custom_node_class.model_fields.get("node_name", type("", (), {"default": node.node_type}))
+        self._add_code(f"# User-defined node: {_node_name_field.default}")
+        self._add_code(f"_custom_node_{node.node_id} = {class_name}()")
+
+        if settings_dict:
+            self._add_code(f"_custom_node_{node.node_id}_settings = {repr(settings_dict)}")
```

Ruff (0.15.18)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)